### PR TITLE
docs(website): updates

### PR DESCRIPTION
**Pull request by hippee-lee**
- This includes Home page fixes for lilia
- Addition of search config to use the new index
- An update for netlify to try to turn of any build actions that trigger
  a yarn install

Signed-off-by: Matt Hippely <mhippely@vmware.com>

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: apps/website/.vuepress/theme/global-components/ReleaseGroup.vue [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <table class="table">
       <tr>
         <th class="left" style="width: 100px;">Scope</th>
+        <th class="left" style="width: 100px;">Project</th>
```
Comment (mathisscott) [inline apps/website/.vuepress/theme/global-components/ReleaseGroup.vue:7]: This should be a rem value (`5rem` iirc)

Px values are not accessible

**Review by mathisscott — CHANGES_REQUESTED** [review]

File: apps/website/.vuepress/theme/global-components/ReleaseGroup.vue (hunk 1 above)
Comment (hippee-lee) [reply apps/website/.vuepress/theme/global-components/ReleaseGroup.vue:7]: You are right, that's what I get for copy paste 😔.

File: apps/website/.vuepress/theme/global-components/ReleaseGroup.vue [hunk 2]
```diff
@@ -3,11 +3,12 @@
     <h3>{{ title }}</h3>
     <table class="table">
       <tr>
-        <th class="left" style="width: 100px;">Scope</th>
-        <th class="left" style="width: 100px;">Project</th>
+        <th class="left fixed-4">Scope</th>
+        <th class="left fixed-4">Project</th>
```
Comment (mathisscott) [inline apps/website/.vuepress/theme/global-components/ReleaseGroup.vue:7]: base font size should be "20px" so 4rem is going to look like 80px.

I assume you've got that covered

**Review by mathisscott — APPROVED** [review]

File: apps/website/.vuepress/theme/global-components/ReleaseGroup.vue (hunk 2 above)
Comment (hippee-lee) [reply apps/website/.vuepress/theme/global-components/ReleaseGroup.vue:7]: thanks. I am still thinking in 24's from the old ways. 
I'll update and may be able to get rid of the second class.

**Review by mathisscott — APPROVED** [review]

**Review by mathisscott — APPROVED** [review]

**Review by gnomeontherun — APPROVED** [review]

**Pull request merged**